lblod/app-subsidie-loket
# Task: remove unneeded services

## ID
DGS-147

## Description

Cleanup the services that are not needed for the subsidy module.

Not everything has been cleaned up in the project, such as:
-  `config/delta` folder, like download-url, prepare-submissions-for-export-service etc.
-  `config/delta-producer/...` folder with unneeded producers (submissions etc.)
-  `config/mock-login/rules.json` unneeded roles.
-  `config/dispatcher/dispatcher.ex` unneeded endpoints.

But these might be better off cleaned up in a different PR.

## Type of change



## How to test

Run the application, visit the subsidy module in the frontend and verify everything still works.
